vitest.config.ts

```diff
@@ -0,0 +1,25 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    globals: true,
+    watch: false,
+    threads: false,
+    environment: 'node',
+    reporters: ['verbose'],
+    coverage: {
+      include: ['lib/**/*.ts'],
+      exclude: [
+        'lib/**/*.spec.ts',
+        'lib/types.ts',
+        'lib/index.ts'
+      ],
+      reporter: ['text'],
+      all: true,
+      lines: 65,
```

Comment:
@kibertoad these numbers are significantly lower than they were in `jest.config.ts`, is this intentional?

Reply:
V8 coverage calculator works differently and reports different numbers, this represents our status quo according to V8. purpose of this PR is not to improve the coverage, we can do that later